README.md

```diff
@@ -59,13 +59,15 @@ grid = ImmersedBoundaryGrid(grid, GridFittedBottom(bathymetry))
 
 # Build an ocean simulation initialized to the ECCO state estimate on Jan 1, 1993
 ocean = ClimaOcean.ocean_simulation(grid)
-dates = DateTimeProlepticGregorian(1993, 1, 1)
-set!(ocean.model, T = ClimaOcean.ECCOMetadata(:temperature; dates),
-                  S = ClimaOcean.ECCOMetadata(:salinity; dates))
+dates = DateTime(1993, 1, 1) : Month(1) : DateTime(1994, 1, 1)
+set!(ocean.model,
+     T=ClimaOcean.Metadata(:temperature; dates=first(dates), dataset=ClimaOcean.ECCO4Monthly()),
+     S=ClimaOcean.Metadata(:salinity;    dates=first(dates), dataset=ClimaOcean.ECCO4Monthly()))
```

Comment:
is there a reason why we can't define an alias `ECCOMetadata`? Not sure why @simone-silvestri decided the syntax change was necessary, but perhaps there is a reason (I don't remember discussing)

Reply:
this is a continue of this discussion, right?
https://github.com/CliMA/ClimaOcean.jl/pull/286#issuecomment-2708961061

Reply:
One approach is that we merge this now so that the code in the example run and discuss design of Metadata later?

Otherwise I can raise an issue and chat there?

Reply:
that works

Reply:
I just noticed https://github.com/CliMA/ClimaOcean.jl/pull/400 and https://github.com/CliMA/ClimaOcean.jl/pull/400#discussion_r1995892703; happy to keep/continue the discussion there or open new issue. I lean for the former.